lib/metrics/context.js

```diff
@@ -194,6 +194,10 @@ module.exports = function (log, config) {
       return logInvalidContext(this, 'expired flowBeginTime')
     }
 
+    if (! HEX.test(metadata.flowId)) {
```

Comment:
Without this, `Buffer.from(flowId, 'hex')` exceptions were being thrown during remote tests, but not from our handlers, so hapi was just reporting it in its weird debug way. Once they were in the same process, I noticed the odd output, and put this fix in.